platform/android/audio_driver_jandroid.cpp

```diff
@@ -79,8 +79,7 @@ Error AudioDriverAndroid::init() {
 	JNIEnv *env = ThreadAndroid::get_env();
 	int mix_rate = GLOBAL_DEF("audio/mix_rate", 44100);
 
-	int latency = GLOBAL_DEF("audio/output_latency", 25);
-	latency = 50;
+	int latency = GLOBAL_DEF("audio/output_latency", 50);
```

Comment:
It looks like this code has been like this forever (since the first commit). I'd say let's just drop the hardcoded `latency = 50` replacement and see how it goes.